examples/networking/xdp/xdp_drop_count.py

```diff
@@ -0,0 +1,135 @@
+#/usr/bin/env python
+#
+# xdp_drop_count.py Drop incomming packets on XDP layer and count for which
+#                   protocol type
+#
+# C code adapted from Brenden Blanco as found in kernel:samples/bpf/xdp1_kern.c
+#
+# Copyright (c) 2016 Jan Rüth
+# Licensed under the Apache License, Version 2.0 (the "License")
```

Comment:
1. I would prefer to have this file checked in as ASL2
2. The original file is GPL (C) PLUMgrid, which you can include as-is but not re-license
3. I can re-license the original file as ASL2, on behalf of PLUMgrid (my employer)
4. Therefore, to achieve 1, I would prefer to be the author of this file, and will include a more appropriate license for the bcc project.
5. You can then later adapt this file, and add your (C), since it will be ASL2 already.